Gemfile.lock

```diff
@@ -158,17 +158,19 @@ GEM
       railties (>= 4.1.0)
       responders
       warden (~> 1.2.3)
-    devise_token_auth (1.1.3)
+    devise_token_auth (1.1.4)
```

Comment:
### [devise_token_auth](http://github.com/lynndylanhurley/devise_token_auth)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.3 → 1.1.4

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **27** commits. See the full changes on [the compare page](https://github.com/lynndylanhurley/devise_token_auth/compare/v1.1.3...v1.1.4).

These are the first 10 commits:
- (2a32f18) [chore(docs): update CHANGELOG.md](https://github.com/lynndylanhurley/devise_token_auth/commit/2a32f18ccce15638a74e72f6cfde5cf15a808d3f)
- (3a34a8d) [chore(locales): add Korean locale (#1346)](https://github.com/lynndylanhurley/devise_token_auth/commit/3a34a8db4368f80782c6c06e62648d5d90678a55)
- (0f09201) [chore(locales): update omniauth.not_allowed_redirect_url for…](https://github.com/lynndylanhurley/devise_token_auth/commit/0f09201e9b86958e2284ab7ffd28b7af8d7c5fee)
- (a93aca6) [repeat any query params after a fragment (#1341)](https://github.com/lynndylanhurley/devise_token_auth/commit/a93aca6922821501950468648deacb48258bd2cc)
- (d6af2a1) [doc: remove duplicated test case on ./docs/usage/testing.md …](https://github.com/lynndylanhurley/devise_token_auth/commit/d6af2a151aacc46c7dd3fb61d122d9b0fdd4019c)
- (d886f47) [feat(oauth-apple): support Sign in with Apple as a documente…](https://github.com/lynndylanhurley/devise_token_auth/commit/d886f476c625a1bf9673f03b95fb629e75ac2875)
- (1987875) [detect Mongoid (till Mongoid will implement it) (#1348)](https://github.com/lynndylanhurley/devise_token_auth/commit/1987875452369882ae9a40b5c9c78e8cd42777ba)
- (e730494) [Fix to be able to use Devise::confirmable module (#1343)](https://github.com/lynndylanhurley/devise_token_auth/commit/e7304946acdbc87935132297d22e186d1acbe2e8)
- (c9bf116) [Fix dead Usage documentation link (#1349)](https://github.com/lynndylanhurley/devise_token_auth/commit/c9bf116735d081ba8888d5098481d23882cd1e65)
- (612af3c) [Please merge again (#1350)](https://github.com/lynndylanhurley/devise_token_auth/commit/612af3c8ba59de230b04216d8a68e3e28695e369)


</details>